doc/2/guides/essentials/installing-kuzzle/index.md

```diff
@@ -208,79 +212,47 @@ Kuzzle uses Elasticsearch and Redis as a persistent and key-value store, respect
 
 Please refer to the [configuration section](/core/2/guides/essentials/configuration) for more details.
 
-### Setup PM2
+### Setup Nodemon
 
-Now that you have installed Kuzzle and loaded its plugins, lets install pm2. Run the following command to install pm2 using NPM as a global package so that it can be run from anywhere on your machine:
+Now that you have installed Kuzzle and loaded its plugins, lets install nodemon. Run the following command to install nodemon using NPM as a global package so that it can be run from anywhere on your machine:
```

Comment:
```suggestion
Now that you have installed Kuzzle and loaded its plugins, let's install nodemon. Run the following command to install nodemon using NPM as a global package so that it can be run from anywhere on your machine:
```